x/pocketcore/types/service.go

```diff
@@ -146,10 +210,18 @@ func (rr RelayResponse) HashString() string {
 }
 
 type relayResponse struct {
-	Signature   string `json:"signature"`
-	Response    string `json:"payload"`
-	RequestHash string `json:"request_hash"`
-	Proof       string `json:"Proof"`
+	Signature string `json:"signature"`
```

Comment:
Why is there a private type that looks exactly like a public type?

Reply:
For overriding json marshalling